Automattic/wp-calypso
# Remove writing prompts setting

This reverts commit 649e4b872ca6af65cf48d7e2007c63e8912e4832, where we added the setting.

The placeholder writing prompt was removed from Jetpack in https://github.com/Automattic/jetpack/pull/28387, and the setting no longer has any effect.

## Testing Instructions

* Go to Settings > Reading in Calypso and verify that you do not see the "Show writing prompts" setting
